indico/modules/rb/client/js/modules/admin/SettingsPage.jsx

```diff
@@ -166,6 +182,19 @@ const SettingsPage = props => {
               description={Translate.string('The maximum length (in days) a booking may last.')}
               validate={v.min(1)}
             />
+            <FinalDropdown
+              options={bookingRequiredOptions}
+              hideValidationError
+              required
+              selection
+              multiple={false}
+              search={false}
+              name="booking_reason_required"
+              label={Translate.string('Booking reason required')}
+              description={Translate.string(
+                'Whether to require to set the reason for a booking or not.'
```

Comment:
"Specify when a booking reason must be provided."?

Reply:
Agreed. It's changed now.